python/ribasim/ribasim/node_types/level_boundary.py

```diff
@@ -1,29 +1,16 @@
 from typing import Optional
 
-import pandera as pa
-from pandera.engines.pandas_engine import PydanticModel
 from pandera.typing import DataFrame
 
-from ribasim import models
 from ribasim.input_base import TableModel
+from ribasim.schemas import (  # type: ignore
```

Comment:
Is this the right place for the comment to apply to both imported schemas? See also other files in this directory if not.

Reply:
It is, otherwise mypy would still complain.